weblogic-azure-aks/src/main/bicep/mainTemplate.bicep

```diff
@@ -46,7 +46,7 @@ param aksClusterRGName string = 'aks-contoso-rg'
 @description('Name of an existing AKS cluster.')
 param aksClusterName string = 'aks-contoso'
 @description('The AKS version.')
-param aksVersion string = 'default'
```

Comment:
I thought we were pulling the version number from the GitHub file. If that's the case, would it not be better to just leave this as default?

Reply:
Hello Edward Burns the customer can not input `default` from UI definition, to keep it consistent, I changed the value with specified version. We can change it to `default`, it will not cause error.

Reply:
Ed has updated the value in https://github.com/oracle/weblogic-azure/pull/140/commits/3cb3a25081f5ac975b8fcb1b3199a8e969e1e332